SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/UnresolvedReferenceGymTest/UnresolvedReferenceGymTest.cpp

```diff
@@ -0,0 +1,57 @@
+
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "UnresolvedReferenceGymTest.h"
+
+AUnresolvedReferenceGymTest::AUnresolvedReferenceGymTest()
+	: Super()
+{
+	Author = "Ajanth";
+	Description = TEXT("Tests what happens when structs with references to actors whose entity have not been created yet are replicated.");
+}
+
+void AUnresolvedReferenceGymTest::PrepareTest()
+{
+	Super::PrepareTest();
+
+	AddStep(TEXT("MakeActorReferences"), FWorkerDefinition::Server(1), nullptr, [this]() {
+		AUnresolvedReferenceGymTestActor* ActorRefsActor =
+			GetWorld()->SpawnActor<AUnresolvedReferenceGymTestActor>({ 0.0f, 0.0f, 0.0f }, FRotator::ZeroRotator);
+		RegisterAutoDestroyActor(ActorRefsActor);
+
+		for (int i = 0; i < 3; i++)
+		{
+			AReplicatedTestActorBase* Actor =
+				GetWorld()->SpawnActor<AReplicatedTestActorBase>(FVector(0, 0, 0), FRotator::ZeroRotator, FActorSpawnParameters());
```

Comment:
Nit: Probably want a consistent style when create a 0,0,0 vector. In this file we have FVector(0, 0, 0) and { 0.0f, 0.0f, 0.0f }. Not sure what the standard is here. Think either is fine.

Reply:
pretty sure there is also an FVector::ZeroVector or something similar

Reply:
`FVector::ZeroVector` exists